app/mm-redux/actions/general.ts

```diff
@@ -73,7 +75,27 @@ export function getDataRetentionPolicy(): ActionFunc {
     return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
         let data;
         try {
-            data = await Client4.getDataRetentionPolicy();
+            const state = getState();
+            const userId = getCurrentUserId(state);
+            const globalPolicy = await Client4.getGlobalDataRetentionPolicy();
```

Comment:
Tested in 5.34, It would return the following object
```
{
    "file_deletion_enabled": false,
    "file_retention_cutoff": 0,
    "message_deletion_enabled": false,
    "message_retention_cutoff": 0
}
```